infra/relayer/migrations/20190620211348-enum-add-gaslimit.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const ENUM_NAME = 'enum_relayer_txn_status'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`ALTER TYPE ${ENUM_NAME} ADD VALUE 'GasLimit';`)
```

Comment:
use ADD VALUE IF NOT EXISTS so that in case we undo the migration it can run again.

Reply:
Considering there's no `down` for this one, I think that's reasonable.